code/API_definitions/quality-on-demand.yaml

```diff
@@ -729,7 +681,8 @@ components:
         * `phoneNumber`
         * `networkAccessIdentifier`
 
-        NOTE: the MNO might support only a subset of these options. The API invoker can provide multiple identifiers to be compatible across different MNOs. In this case the identifiers MUST belong to the same device
+        NOTE1: the MNO might support only a subset of these options. The API invoker can provide multiple identifiers to be compatible across different MNOs. In this case the identifiers MUST belong to the same device
```

Comment:
Can we remove MNO, since it is mobile specific?

Reply:
Sure, it's anyway part of the alignment with Commonalities 0.4.0-rc.2 alignment. Done in https://github.com/camaraproject/QualityOnDemand/pull/326/commits/70ea70ee9386eebd4b7e102887021739cedb1385